public/i18n/enumKeyLookups.js

```diff
@@ -0,0 +1,20 @@
+export default {
```

Comment:
Need to update filters.acceptItems to match these new values? i.e., 'overalls' will match both Medical overalls and Coveralls/bunny suits. 'booties' will match both Medical and Disposable booties. Masks will match both Surgical and Homemade. etc. etc.

Reply:
hmm ok. I think I need to talk to someone a bit about the filters. I'll post in data, I think they've been updating the forms and it's worth a revisit on what the filters should be and how they should match

Reply:
I'm wondering about the open packages choices. You have yes and no, but what about "other" on the form.  Or should there be an "other" on the form?

Reply:
There is an "other" category for both the "items accepted" and the "open packages" choices however I think it is less important to translate those.

This is the current Google Forms process:

1. User submits form in some language (e.g. French)
2. AppScript converts the "enum" categories to English so that filtering can be applied in a single language
3. Sheet written to data.json file
4. App loads sheet and renders it

What we're doing here is translating all the "enum" categories that we know are options. We could do the other columns but we'd need to use something like Google Translate so for now I think it's best to just leave the freeform fields as is